apps/collabora_docker.sh

```diff
@@ -227,6 +230,12 @@ then
   SSLProxyVerify None
   SSLProxyCheckPeerCN Off
   SSLProxyCheckPeerName Off
+  
+  # Improve security settings
+  Header set X-XSS-Protection "1; mode=block"
+  Header set Strict-Transport-Security "max-age=31536000; includeSubDomains; preload"
+  Header set X-Content-Type-Options nosniff
+  Header set Content-Security-Policy "frame-ancestors 'self' ${NCDOMAIN//\\/}"
```

Comment:
I'll add this comment so that we don't forget to test this.

Reply:
Except this, is everything else tested?

Reply:
Just had a quick look, menus looks good. Haven't tested the scripts but I guess you copied from old (working) stuff?

Reply:
Yes, I copied most things from old (working) stuff 👍